engine/orchestration/src/com/cloud/agent/manager/AgentManagerImpl.java

```diff
@@ -242,11 +239,11 @@ public boolean configure(final String name, final Map<String, Object> params) th
     }
 
     protected int getPingInterval() {
-        return PingInterval.value();
+        return ManagementServiceConfiguration.PingInterval.value();
     }
 
     protected long getTimeout() {
```

Comment:
Since you've introduced a new `ManagementServiceConfiguration`, why not refactor/move this method to the class and make it reusable across server/codebase.